src/js/utils/common.js

```diff
@@ -76,7 +76,6 @@ export function generateVirtualApiVersions() {
 }
 
 // TODO: these are temp binding while transition to module happens
-window.microtime = microtime;
```

Comment:
I suppose this is not related to this PR and this is being fixed here: https://github.com/betaflight/betaflight-configurator/pull/2414

Reply:
true, but I can't just push broken code (as I did). So will rebase when that is done. But till then I'll keep this here.